### DISCUSSION TITLE
Profile management, it it possible?

### DISCUSSION THREAD

**Discussion by meluiz** [zed-industries/zed#12098] [Feature Requests]
I work with mobile and front-end development and sometimes play with backend (using JavaScript) and have different configurations for each stack. Is it possible, and if not, are there plans for profile management?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
